test/unit/start_key_test.lua

```diff
@@ -0,0 +1,29 @@
+local expirationd = require('expirationd')
+local t = require('luatest')
+local g = t.group("start_key")
```

Comment:
The same as https://github.com/tarantool/expirationd/pull/52/commits/cb50f28c417266ea07c391c31f4eeeb6943e97cb#r656454532